wellplated/models.py

```diff
@@ -142,66 +142,67 @@ class Container(ClusterableModel):
     panels = (
         FieldPanel('created_at', read_only=True),
         FieldPanel('format', read_only=True),
-        InlinePanel('wells'),
+        InlinePanel('positions'),
     )
 
-    wells: Manager['Well']
+    positions: Manager['Position']
```

Comment:
Is `Position` here a string literal, or a type?

Reply:
The class is defined later on line 185 so the reference here uses the [quoted forward reference style](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#forward-references). Maybe I should try out the other styles like `from __future__ import annotations`. https://mypy.readthedocs.io/en/stable/runtime_troubles.html#forward-references additional reading

Reply:
Yeah I am familiar with the quoted reference style, I just didn't realized it applied to `Generic`s too. TIL!